src/src/com/microsoft/aad/adal/CacheKey.java

```diff
@@ -155,6 +155,14 @@ public static String createMultiResourceRefreshTokenKey(AuthenticationRequest it
         return createCacheKey(item.getAuthority(), item.getResource(), item.getClientId(), true,
                 cacheUserId);
     }
+    
+    public static String createFamilyRefreshTokenKey(final AuthenticationRequest authRequest, final String userId) {
+        if (authRequest == null) {
+            throw new IllegalArgumentException("authentication request is null");
+        }
+        
+        return createCacheKey(authRequest.getAuthority(), null, null, true, userId);
```

Comment:
As we discussed, i'll keep it as a string. When we get family token back, i'll use whatever server returns us and store it as foci-serverReturnedFamilyId as client id. For lookup, will still hard coe the familyid as "1"